StripeFinancialConnections/StripeFinancialConnections/Source/Web/FinancialConnectionsWebFlowViewController.swift

```diff
@@ -416,3 +420,20 @@ private extension FinancialConnectionsWebFlowViewController {
         return startPollingParam + "&\(fragment)"
     }
 }
+
+extension FinancialConnectionsWebFlowViewController {
+    static func updateAdditionalParameters(
+        startingAdditionalParameters: String?,
+        isInstantDebits: Bool,
+        linkMode: LinkMode?
+    ) -> String? {
+        var additionalQueryParameters = startingAdditionalParameters
+        if isInstantDebits {
+            additionalQueryParameters = (additionalQueryParameters ?? "") + "&return_payment_method=true"
```

Comment:
Nit: Should we just do the nil coalescing once at the start of this method? It would mean that we always return a non-nil string, but that seems fine to me.

Reply:
Done! https://github.com/stripe/stripe-ios/pull/4057/commits/5a1b90f8d43a8e216ea615c2ccceb3d039421d1b Worked out pretty nicely!